jwst/regtest/test_fgs_image2.py

```diff
@@ -25,5 +25,10 @@ def test_fgs_image2(run_fgs_image2, rtdata_module, fitsdiff_default_kwargs, suff
 
     rtdata.get_truth(f"truth/test_fgs_image2/{output}")
 
+    # Adjust tolerance for machine precision with float32 drizzle code
+    if suffix == "i2d":
+        fitsdiff_default_kwargs["rtol"] = 3e-3
```

Comment:
Wow, the differences can be that big?  If I'm reading this right, that's 0.3%, which is a lot (IMHO)!

Reply:
Yes.  This is why it is important to get the C drizzle code to use `float64` instead of `float32`.